### DISCUSSION TITLE
Github education benefits

### DISCUSSION THREAD

**Discussion by shubhlpu16** [community/community#144502] [GitHub Education]
### Hi everyone,

I am approved on 11 nov. Still i cant access benefits

**Comment by OctavianAdv** [comment]
Hi @shubhlpu16 

When the benefits become available, the status label will turn purple and change to:

Congrats! Your request for @shubhlpu16 was approved and your benefits are now available.
The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions:
https://github.com/orgs/community/discussions/132790 , https://github.com/orgs/community/discussions/131665 , https://github.com/orgs/community/discussions/132470 , https://github.com/orgs/community/discussions/133531, https://github.com/orgs/community/discussions/134279, https://github.com/orgs/community/discussions/135849, https://github.com/orgs/community/discussions/136887, https://github.com/orgs/community/discussions/138721, https://github.com/orgs/community/discussions/139268, https://github.com/orgs/community/discussions/140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
